sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -39,7 +39,21 @@ import org.apache.spark.util.Utils
 
 
 /**
- * Functions available for DataFrame operations.
+ * Commonly used functions available for DataFrame operations. Using functions defined here provides
+ * a little bit more compile-time safety to make sure the function exists.
+ *
+ * Spark also includes more built-in functions that are less common and are not defined here.
+ * You can still access them (and all the functions defined here) using the [[functions.expr()]] API
+ * and calling them through a SQL expression string. You can find the entire list of functions for
+ * the latest version of Spark at [[https://spark.apache.org/docs/latest/api/sql/index.html]].
```

Comment:
@rxin, it's rather a nit but shouldn't we always update the link for each release since it always points the latest?

Reply:
it's just a lot of work and i'm sure we will forget to update ... so i'm pointing to the latest.